src/components/Forms/create-repair-request.tsx

```diff
@@ -0,0 +1,169 @@
+      tncAccepted: false
+    }
+  });
+
+  const { data: itemTypeList } = useItemTypes();
+  const { data: brandList } = useBrands();
+  const { data: eventOptions } = useEventOptions();
+  const { mutate: createRepairRequest } = useCreateRepairRequest();
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    const uploadPromises =
+      data.images?.map((image) => uploadImage(image)) ?? [];
+    const uploadThumbailPromise = data.images?.[0]
+      ? generateThumbnail(data.images[0]).then(uploadImage)
+      : undefined;
+    const keys = await Promise.all([uploadThumbailPromise, ...uploadPromises]);
+    const [thumbnailImage, ...images] = keys;
+    const updatedData = { ...data, thumbnailImage, images };
+    createRepairRequest(updatedData);
+  };
+
+  return (
+    <div className="flex items-center justify-center p-4">
+      <div className="flex w-screen flex-col justify-center gap-4 md:w-3/6 lg:w-full">
+        {/* Logo of Repair Lab, which links to the main website. */}
+
+        <picture className="flex justify-center">
+          <a href="https://repairlab.myfreesites.net/" target="_blank">
+            <Image
+              src="/images/repair_lab_logo.jpg"
+              alt="Repair Labs Logo"
+              width={80}
+              height={80}
+            />
+          </a>
+        </picture>
+
+        {/* Heading of the Page */}
+
+        <h1 className="flex justify-center text-xl font-bold">
+          {" "}
+          Add a repair request
+        </h1>
+        <form onSubmit={handleSubmit(onSubmit)} className="flex flex-col gap-4">
+          {/* Input field for Brand of Item */}
+
+          <SingleSelect
+            name="itemBrand"
+            control={control}
+            placeholder="Select a brand"
+            label="Brand"
+            rules={{ required: true }}
+            options={
+              brandList
+                ? brandList.map((brand: Brand) => {
+                    return { id: brand.name, text: brand.name };
+                  })
+                : []
+            }
+          />
+
+          <SingleSelect
+            name="itemType"
+            control={control}
+            placeholder="Select an item type"
+            label="Item Type"
+            rules={{ required: true }}
+            options={
+              itemTypeList
+                ? itemTypeList.map((itemType: ItemType) => {
+                    return { id: itemType.name, text: itemType.name };
+                  })
+                : []
+            }
+          />
+
+          {/* Input field for Description of Item */}
+
+          <FieldTextArea
+            name="description"
+            label="Description"
+            placeholder="Enter a description"
+            control={control}
+            rules={{ required: true }}
+          />
+
+          <FieldImageUpload multiple name="images" control={control} />
+
+          {/* Input field for Event Date */}
+          <SingleSelect
```

Comment:
There's no need to render this on the event page because it should get that info from its props. Consider adding an optional prop to this form called `eventId`, then rendering it based on whether it's passed to the component. Then you can use the `eventId` in the POST request.